content/webapp/components/ImageCard/index.tsx

```diff
@@ -48,6 +50,7 @@ const ImageCard: FunctionComponent<Props> = ({
         }}
         id={id}
         data-gtm-trigger={isEnhanced ? 'open_image_modal' : undefined}
+        data-gtm-position-in-list={isEnhanced ? positionInList : undefined}
```

Comment:
Why is it undefined if no JS? (Analytics wouldn't work anyway, right?)

Reply:
ha, of course I put it in initially and [Gareth questioned it](https://github.com/wellcomecollection/wellcomecollection.org/pull/9562/files#r1159907975). Now, years later, I am ready to accept my mistake. I'll update the trigger too.